src/mint/library.cairo

```diff
@@ -518,35 +625,37 @@ namespace CarbonableMinter {
             assert_not_zero(caller);
         }
 
-        // [Check] Desired quantity is lower than maximum allowed per transaction
-        let (max_buy_per_tx) = CarbonableMinter_max_buy_per_tx_.read();
-        let quantity_allowed = is_le(quantity, max_buy_per_tx);
-        with_attr error_message("CarbonableMinter: quantity not allowed") {
-            assert quantity_allowed = TRUE;
+        // [Check] Desired amount is within range
+        let (max_value_per_tx) = CarbonableMinter_max_value_per_tx_.read();
+        let (min_value_per_tx) = CarbonableMinter_min_value_per_tx_.read();
+        with_attr error_message("CarbonableMinter: amount not allowed") {
+            assert_le(min_value_per_tx, amount);
+            assert_le(amount, max_value_per_tx);
```

Comment:
`assert_in_range` ?

Reply:
You got me, in the end I didn't go for it because of the need of +1 here

Reply:
Same here, readibility vs. optimization, since starknet gas for computation should be very low, I would prefer readibility vs. optimization (once again, if the optimized gap is "acceptable").